### DISCUSSION TITLE
Is a resistor value change needed...

### DISCUSSION THREAD

**Discussion by gsargent70** [frntc/Sidekick64#97] [Q&A]
If I put on a 3mm LED on position one and two of the four LEDs my sidekick64 board will not boot.  Would a resistor value change be needed.  The 3mm LED's I'm trying to use are 2-2.2V 20mA and the SMD ones pcbway put on the board are 1.6-2.6V 20mA so I would think no change in resistor value needed.  With the SMD ones on it works just fine.

**Comment by frntc** [comment]
Did you remove the SMD LEDs at position one and two?

**Reply by gsargent70** [reply]
Yes

**Reply by frntc** [reply]
Hm, I don't see a reason why the SK64 should not boot. There's nothing on the TFT and also not on HDMI? And if you remove the LED everything works again? (if the answer to the last question is yes, can you test the LED without SK64?)
